### ISSUE TITLE
Component does not always emit correct page number

### ISSUE THREAD

**Issue by domdeger** [stephanrauh/ngx-extended-pdf-viewer#147]
**Check this first**
- Does the bug also happen on https://mozilla.github.io/pdf.js/web/viewer.htm? NA
- Does the bug also happen on https://pdfviewer.net/simple? NA

**Describe the bug**

**Version info**
- Version of ngx-extended-pdf-viewer 1.5.2
- Version of the JavaScript files pdf.js, pdf-worker.js and viewer.js (currently: default files vs. version 2.2 preview files)

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Windows 10 1903
 - Browser [e.g. chrome, safari] Chrome 
 - Version [e.g. 22] 77

**Smartphone (please complete the following information):**

**To Reproduce**
Load http://www.connectedthebook.com/pdf/excerpt.pdf in component and subscribe to pageChange and scroll through the first pages.

**Additional context**
Proposed a fix at https://github.com/stephanrauh/ngx-extended-pdf-viewer/pull/146

**Comment by stephanrauh** [comment]
Interesting. That's a scenario I wasn't aware of. Thanks for pointing it out and for providing a pull request!

Currently, I've got very little free time, but I'll review and - hopefully - integrate your PR ASAP. Probably in a week or so.

Best regards,
Stephan

**Comment by stephanrauh** [comment]
I think we can allow Roman page numbers (or arbitrary strings), too. I'm currently trying to extend the PDF viewer to achieve this goal.

**Comment by stephanrauh** [comment]
I've published your PR with version 1.5.3. However, there's still work to be done. It seems that `(page)` receives the "page label" - i.e. the page number as defined by the author. On the other hand, `[page]` refers to the page number, which always starts at 1 and which is always numeric. Obviously, that's an unhealthy mixture of two different concepts. I plan to fix that with version 1.6.0.

**Comment by stephanrauh** [comment]
@domdeger (and everyone who's reading this): I believe that introducing the new attribute `[(pageLabe)]` is a breaking change, especially since I've modified the internal API slightly. So I'd appreciate your help. Would you like to have a look at the new preview version 1.6.0-rc.0 I've just published on npm?

Thanks in advance,
Stephan

**Comment by domdeger** [comment]
@stehphanrauth thanks for investigating. I agree with you that the component should inform about the Page definied by the author and also about the page Index or just a numerical page value. 

In my use case I want top store the current Page so I the User can continue reading when coming back. 

I will review the preview next week and give feedback since i am a bit busy lately.

Best regards,
Dominik

**Comment by stephanrauh** [comment]
I've shipped the feature with 1.6.0. So I'm closing this ticket now. If you discover a bug, don't hesitate to report it here: I'm still listening.

There's one thing I'm uncertain of. Most PDF document don't have page labels. They only exist if the author assigns them manually. I wonder I the PDF viewer should use the page number as default. As of version 1.6.0, I've decided not to do so. Removing synthetic page labels is a bigger breaking change as adding them.

If anybody thinks that default page labels are a good idea: I'd like to hear from you!
